src/blockchain-scanner.service.ts

```diff
@@ -26,6 +26,11 @@ export class BlockchainScannerService implements OnApplicationBootstrap {
     // Kick off initial scan
     const initialTimeout = setTimeout(() => this.scan(), 0);
     this.schedulerRegistry.addTimeout('initialScan', initialTimeout);
+
+    const graphUpdateQueueIsPaused = await this.graphUpdateQueue.isPaused();
+    if (graphUpdateQueueIsPaused) {
+      this.graphUpdateQueue.resume();
+    }
```

Comment:
I don't think we want to categorically resume the queue at startup. We need to check Capacity first--if we're at our limit, we need to defer queue resumption until the next epoch.

Reply:
Do we want to push this logic to #61 @JoeCap08055 ?